py_zipkin/zipkin.py

```diff
@@ -112,6 +111,7 @@ def __init__(
         use_128bit_trace_id=False,
         host=None,
         context_stack=None,
+        span_store_stack=None,
```

Comment:
reading this again, i'd like to bikeshed the name a bit. There's no reason this has to be a stack, so maybe we can call it something different?